icicle/curves/bls12_377/params.cuh

```diff
@@ -142,6 +142,7 @@ namespace PARAMS_BLS12_377 {
 
   struct fq_config {
     static constexpr unsigned limbs_count = 12;
+    static constexpr unsigned omegas_count = 48;
```

Comment:
This is nitpicky, but it looks like this was manually changed? Do we want to also change this [here](https://github.com/celer-network/icicle/blob/main/curve_parameters/bls12_377.json#L6)

Reply:
This is actually a situation not accounted for by the script. You can notice that `fq_config` is not the scalar field, but the base field. And the script only allows for NTT parameters in the scalar field because most of the time, scalar field is the only one we compute NTTs over. BLS12-377 is special because its base field is designed to be the scalar field of another curve - BW6-671. That's why `omegas_count` and twiddles have been added to the base field by hand here.
An alternative option would be to repeat this field in the BW6-671 config file which would align the codebase with the script better. But we decided not to do this to avoid code duplication, plus I think having one less field would decrease compilation time and the size of our static lib at least a bit.